### PR TITLE
Remove the ghetto REST flash-based video player

### PR DESCRIPTION
**Pull request by jvoisin**
Remove the terrible REST flash-based video player. This includes flash files, related javascript, removing the controller, the jsp file, and purging every single trace of Flash-related code from airsonic!

### CODE REVIEW

**Comment by jvoisin** [comment]
This should close #1098

File: airsonic-main/src/main/java/org/airsonic/player/security/GlobalSecurityConfig.java [hunk 1]
```diff
@@ -162,7 +162,7 @@ protected void configure(HttpSecurity http) throws Exception {
                     .and().authorizeRequests()
                     .antMatchers("/recover*", "/accessDenied*",
                             "/style/**", "/icons/**", "/flash/**", "/script/**",
-                            "/sonos/**", "/crossdomain.xml", "/login", "/error")
```
Comment (muff1nman) [inline airsonic-main/src/main/java/org/airsonic/player/security/GlobalSecurityConfig.java]: This is unrelated to this PR, correct?

Comment (jvoisin) [reply airsonic-main/src/main/java/org/airsonic/player/security/GlobalSecurityConfig.java]: Nope, it's related: since we're not using Flash anymore, we can remove the crossdomain.xml file.

**Comment by eharris** [comment]
@jvoisin Thanks for this!  But it looks like the internationalization strings still reference the "Flash" player, which should probably be changed to something more generic like "Media" or maybe HTML5?

**Comment by jvoisin** [comment]
Done in 2d06995416f589ad9664b4ca1d03963326b3ca0d
